### ISSUE TITLE
Be clearer about forcing people to add SSL when using a domain name

### ISSUE THREAD

**Issue by fharper** [meilisearch/meilisearch-digitalocean#73]
I thought I could add my domain name to my MeiliSearch instance without SSL. I know it's not a good practice, SSL ALL THE THINGS, but it was only for a quick test in which I don't care someone steal the data between the server and my computer.

It would be nice to specify it in the process, as I said no to certbot, my own certificate, and arrive to a step where I needed to give the pem file. Having know we insist on having SSL, I would have accept the certbot option because I'm lazy :)

Obviously I can just run the script again, but I believe in the little things we can improve :D

(or maybe we don't enforce SSL? I mean, at that point, if they say no, it's their server their choice, not ours)

![image](https://user-images.githubusercontent.com/1170989/102274335-b6368980-3ef1-11eb-9d3f-b2a06b77c6cb.png)

**Comment by eskombro** [comment]
This seems to be a BUG.

You should be able to do this. If you said no to certbot and no to your own SSL, that's on you, we respect your decision :D
So it shouldn't be asking for your `.pem`

I'll dig into it!

**Comment by eskombro** [comment]
@fharper 

This issue is solved in #69 but won't be yet submitted to production. If you want to fix it in your own droplet, you can ssh on it and replace in the file

this line:
```bash
if [ "$USE_SSL" = "" ]; then
```

with this
```bash
if [ "$USE_SSL" = false ]; then
```

Sorry about that 👍🏽
